14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Get started with the Management API.
+---
+
+# Management API
+
+The Management API delivers headless capabilities built directly into Umbraco. Unlike the Content Delivery API, the Management API enables interaction with everything else in the backoffice except for the Content and Media items.
+
+The API returns responses in a JSON format, which can be presented in different channels using your preferred technology stack. This feature maintains the user-friendly editing experience of Umbraco, while also ensuring an efficient delivery of content in a headless manner. Additionally, with its different extension points, the API can be customized to meet a wide range of requirements.
```

Comment:
I would skip this section. It sounds like something describing the Delivery API. The Management API is for managing, editing, handling, whatever ---- not for "delivery of content" 😄